src/Board.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Board.hpp"
+
+
+Board::Board()
+    : field(40)
+{}
+
+int Board::getNumberOfFields() const noexcept
+{
+    return numberOfFields;
+}
+
+int Board::getField(const int &numberOfField) const noexcept
```

Comment:
In my opinion it should be `Field` as return type instead `int`. Could you tests this method, please.